### ISSUE TITLE
`ocrd resmgr download ocrd-tesserocr-recognize <model>` is broken: tries to download from old master branch

### ISSUE THREAD

**Issue by twerkmeister** [OCR-D/core#719]
Using latest `ocrd-all:maximum` docker container ✌️ 

To reproduce download some training data you don't already have:
`ocrd resmgr download ocrd-tesserocr-recognize Latin.traineddata`

you'll see in the log 
`INFO ocrd.resource_manager._download_impl - Downloading https://github.com/tesseract-ocr/tessdata_fast/raw/master/script/Latin.traineddata to /usr/local/share/ocrd-resources/ocrd-tesserocr-recognize/Latin.traineddata`

and just after that 

```
Traceback (most recent call last):
  File "/usr/bin/ocrd", line 33, in <module>
    sys.exit(load_entry_point('ocrd', 'console_scripts', 'ocrd')())
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/dist-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/build/core/ocrd/ocrd/cli/resmgr.py", line 144, in download
    progress_cb=lambda delta: bar.update(delta)
  File "/build/core/ocrd/ocrd/resource_manager.py", line 210, in download
    self._download_impl(url, fpath, progress_cb)
  File "/build/core/ocrd/ocrd/resource_manager.py", line 161, in _download_impl
    total = size if size else int(r.headers.get('content-length'))
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```
as the file didn't exist. Used
`wget https://github.com/tesseract-ocr/tessdata_fast/raw/main/script/Latin.traineddata` and moved it to the location from the log `/usr/local/share/ocrd-resources/ocrd-tesserocr-recognize/Latin.traineddata`

Now it works

**Comment by hnesk** [comment]
Fixed in #721 + #726

**Comment by kba** [comment]
Fixed in https://github.com/OCR-D/core/releases/tag/v2.26.1
